### ISSUE TITLE
Support Allium Again?

### ISSUE THREAD

**Issue by mikerodey** [technobyl/CryptoDredge#79]
Garlicoin interest has been high lately, some users are trying to use 30xx cards but these cards are not supported in older versions of CryptoDredge that used to support Allium.

**Comment by protox** [comment]
Yeah we are trying to get to Mars here and maybe beyond and CryptoDredge seems to give us a good hash rate, so would be pretty cool if latest versions reintroduced allium. Thanks in advance.

**Comment by vomar** [comment]
Garlicoin is coming back into favor, you could add it back, I myself have an RTX 3090 and would like to mine garlicoin on it.

**Comment by Starec666** [comment]
My friends with 30xx are unhappy not to mine Garlicoin, while I do it with older crypto dredge and 1080ti )

**Comment by nn47nn** [comment]
Действительно братан, люди по делу пишут, аллиум погорячился удалять, да и для новвых карточек его бы запилить и вобще нормалёк.

**Comment by Urgo** [comment]
bump?
